### ISSUE TITLE
Default cmd output formatting

### ISSUE THREAD

**Issue by ydyote** [Exiv2/exiv2#767]
Hello, I have only one feature request and that is to improve the default output of meta information
The default should look like following:

TagDescription [TagName] : value [valueWhichCanBeSet]

basically to offer human readable tags and descriptions with the actual tags and their "raw" values

I would also provide an example but I did not use this tool properly yet to be able to provide it, because I don't have time now to somehow guess the correct formats (raw values) for any tags

Sorry for not being completely clear, I hope it will be understood.
If this is somehow configurable/achievable I'm really sorry for bothering you.

Tool seems to be pretty cool.

**Comment by D4N** [comment]
> I would also provide an example

That would be quite helpful, since I don't really have an idea what you'd wish the output to be.

**Comment by ydyote** [comment]
Ok, it should look something like this:

Exposure Time [ExposureTime] : 1/1300 s [1/1300]
Aperture [FNumber] : F5 [5.0]

Maybe its just me but I have no idea how to gather necessary information easily in this use case:
I can see exif metadata in the windows explorer file properties window, there are listed formatted descriptions and values of properties.. not all of them can be changed but this is possible by other tools..

for those tools (I think also for this one, I'm sorry if I'm incorrect haven't verify this yet) I have to specify "tag name" and proper value (which is not formatted the same way as the windows file properties window displays them)
So basically this should help me (and other people) to make a list of necessary things I need to set because I know the what kind of description I'm looking for, the tag name will be displayed in [ ] and the value for the tag will be displayed in the same row too formatted and unformatted

I hope it is more clear now and you will consider this feature, thank you for your time

**Comment by clanmills** [comment]
We are not changing this.  Exiv2 has used the current output scheme for 15 years and many folks have scripts which parse this output.  There are many options provided from the exiv2 command-line to control output.  Please see the `-P  flg		--Print      Print report (fine grained control)` in the man page.  We provide both "raw" (-pv) values and "translated" human readable values (-pt).

**Comment by ydyote** [comment]
Well it doesn't need to be the default scheme but maybe add this as an option. I tried the -pv and -pt, but still there is no way to display all necessary information for a normal user, only for people who have the patience to go through the naming hell - because a "noob" user has to take at least three different outputs and put them together to get the complete and valuable information
outputs from: command without any settings, command with pv and command with pt.

I did not find any other metadata tool which is powerful enough (or even somewhat useful, it is open source and it is not a one man show), too bad that this one is not more user friendly (or does not have a simple option to print the information in some more useful way like some flag e.g. -prettyprint)

Keep up the good work. Thanks.
